e2e-tests/constants/examples/companies-house-number-examples.js

```diff
@@ -1,4 +1,5 @@
 export const COMPANIES_HOUSE_NUMBER = '08989898';
+export const COMPANIES_HOUSE_NUMBER_COMPANY_WITH_SPECIAL_CHARACTER_NAME = '00947741';
```

Comment:
`COMPANIES_HOUSE_NUMBER_SPECIAL_CHARACTER/S_NAME`? More aligned with e.g `COMPANIES_HOUSE_NUMBER_MULTIPLE_SIC_CODES`


Does the company name have multiple special characters?

Reply:
Just one